benchmark/checkperf.cmake

```diff
@@ -7,7 +7,8 @@
 
 # Clone the repository if it's not there
 find_package(Git QUIET)
-if (Git_FOUND)
+if (SIMDJSON_IS_UNDER_GIT AND Git_FOUND AND (GIT_VERSION_STRING  VERSION_GREATER  "2.1.4")) # We use "-C" which requires a recent git
```

Comment:
sorry sorry sorry one space between GIT_VERSION_STRING and VERSION_GREATER sorry sorry sorry

Reply:
Fixed.